src/main/java/com/connectfour/ConnectFour.java

```diff
@@ -1,18 +1,43 @@
 package com.connectfour;
 
+import java.util.Scanner;
+
 public class ConnectFour {
+
+    private static Display display;
+    private static Scanner scanner;
+    private static PlayerScanner playerScanner;
+    private static Board board;
+    private static Disc disc1;
+
+    private static Player playerOne;
+
+
     public static void main(String[] args){
-        Display display = new Display(System.out);
-        display.println("Connect Four!\n");
+        setUpGame();
 
-        Disc testDisc = new Disc("red", '\u25C9');
-        Disc otherDisc = new Disc("blue",'\u25C9' );
-        Board board = new Board();
+        display.println("Welcome to Connect Four!!\n");
 
         Presenter.printBoard(display, board);
 
-        board.addDisc(testDisc, 0 , 0);
-        board.addDisc(otherDisc, 3 , 4);
+        nextTurn(playerOne);
+
         Presenter.printBoard(display, board);
     }
+
+    private static void setUpGame(){
+        display = new Display(System.out);
+        scanner = new Scanner(System.in);
+        playerScanner = new PlayerScanner(scanner);
+        disc1 = new Disc("red", '\u25C9');
+        board = new Board();
+        playerOne = new Player("Player One", disc1, playerScanner);
+    }
+
+    private static void nextTurn(Player player){
+        display.println(player.name + ", please select which column you would like to place your disc.");
+        display.println("Your color is " + player.disc.getColor());
```

Comment:
Let's store the strings as static variables in this class! (since they won't be used anywhere else)

Reply:
Kk, will do!